test/register_cluster1_cluster2_with_ocm.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+# Copyright 2024 The KubeStellar Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+echo "Starting OCM registration"
+curl -L https://raw.githubusercontent.com/open-cluster-management-io/clusteradm/main/install.sh | bash
```

Comment:
So this is not just registering a cluster, I see installation of  `clusteradm` here too. I think this should be broken out into a separate step, and avoid repeating the installation if the command is already there (possibly insisting on an acceptable version).

Reply:
The ocm installation should really be a pre-req step.  For now I moved it into its own script.